aws/resource_aws_dynamodb_table_test.go

```diff
@@ -950,53 +957,23 @@ resource "aws_dynamodb_table" "basic-dynamodb-table" {
 }`, rName)
 }
 
-func testAccAWSDynamoDbConfigStreamSpecification() string {
+func testAccAWSDynamoDbConfigStreamSpecification(enabled bool, viewType string) string {
 	return fmt.Sprintf(`
 resource "aws_dynamodb_table" "basic-dynamodb-table" {
   name = "TerraformTestStreamTable-%d"
   read_capacity = 10
   write_capacity = 20
   hash_key = "TestTableHashKey"
-  range_key = "TestTableRangeKey"
```

Comment:
Thank you for making the configuration more relevant to what we're trying to test here 👍